test/partitiontest/filtering.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2019-2021 Algorand, Inc.
```

Comment:
Rename this to `filtering_test.go` so that it isn't included in production binaries.